web.tf

```diff
@@ -39,6 +39,9 @@ resource "aws_security_group" "acme_web" {
     from_port   = 137
     protocol    = "tcp"
     cidr_blocks = ["192.164.0.0/24"]
+    
```

Comment:
**Accurics remediation**
 
**[HIGH]: http port open to internet**
 
**Vulnerability**
http port open to internet
 
**Remediation**
Change cidr_block to non 0.0.0.0/0